### PR TITLE
For #180: Adding more ITs for RtContainer.

### PR DESCRIPTION
**Pull request by bkuzmic**
For #180:

- adding more ITs for RtContainer to cover: stop, restart, kill methods.
- for Travis CI: pull ubuntu Docker image to enable RtContainer ITs

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#216](https://github.com/amihaiemil/docker-java-api/issues/216) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#216](https://github.com/amihaiemil/docker-java-api/issues/216) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/2e589694-af27-4380-826a-10dec3e024d8); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/com/amihaiemil/docker/RtContainerITCase.java [hunk 1]
```diff
@@ -58,4 +62,80 @@ public void renamesContainer() throws Exception {
...
     }
+
+    /**
+     * {@link RtContainer} can start/stop Docker container it represents.
+     * @throws Exception If something goes wrong.
+     */
+    @Test
+    public void startStopContainer() throws Exception {
+        final Container container = new LocalDocker(
+            new File("/var/run/docker.sock")
```
Comment (paulodamaso) [inline src/test/java/com/amihaiemil/docker/RtContainerITCase.java]: @bkuzmic These path does not works on windows machines.

File: src/test/java/com/amihaiemil/docker/RtContainerITCase.java [hunk 2]
```diff
@@ -58,4 +62,80 @@ public void renamesContainer() throws Exception {
...
+        ).containers().create("TestStart", this.containerJsonObject());
+        container.start();
+        MatcherAssert.assertThat(
+            container.inspect().getJsonObject("State").getBoolean("Running"),
```
Comment (paulodamaso) [inline src/test/java/com/amihaiemil/docker/RtContainerITCase.java]: @bkuzmic Maybe it will be better if we extract  "Running" value to a constant field.

Comment (bkuzmic) [reply src/test/java/com/amihaiemil/docker/RtContainerITCase.java]: @paulodamaso I've extracted this call to a separate method. It looks cleaner to me. What do you think?

File: src/test/java/com/amihaiemil/docker/RtContainerITCase.java (hunk 1 above)
Comment (amihaiemil) [reply src/test/java/com/amihaiemil/docker/RtContainerITCase.java]: @paulodamaso no need for puzzles, I think. I would say that we assume by default that ``LocalDocker`` will only work on linux -- it uses ``Unix Sockets`` behind, which I don't think are available on Windows :)

Comment (paulodamaso) [reply src/test/java/com/amihaiemil/docker/RtContainerITCase.java]: @amihaiemil Okay then, that's good for me

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by 0crat** [comment]
Job was finished in 25 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**